modules/ossm-deploy-cluster-wide-control-plane-cli.adoc

```diff
@@ -12,6 +12,7 @@ You can configure the `ServiceMeshControlPlane` resource for cluster-wide deploy
 
 * The {SMProductName} Operator is installed.
 * You have access to the OpenShift CLI (`oc`).
+* There is no other `ServiceMeshControlPlane` object in the cluster.
```

Comment:
**QE PTAL**

Reply:
Add this Parameter to SMCP:

mode: ClusterWide in SMCP yaml, then you can deploy the Cluster-wide 

This is the error:

```
Fix the following errors:
admission webhook "smcp.validation.maistra.io" denied the request: a cluster-scoped SMCP may only be created when no other SMCPs exist
```

If, we delete the existing SMCP, then we can create the Cluster-wide SMCP.